subsys/bluetooth/controller/ll_sw/ull_scan.c

```diff
@@ -175,7 +232,32 @@ uint8_t ll_scan_enable(uint8_t enable)
 		ull_filter_rpa_update(false);
 		lll->rpa_gen = 1;
 	}
-#endif
+#endif /* CONFIG_BT_CTLR_PRIVACY */
+
+#if defined(CONFIG_BT_CTLR_ADV_EXT)
+#if defined(CONFIG_BT_CTLR_PHY_CODED)
+	if (!is_coded_phy || (scan->lll.phy & BT_HCI_LE_EXT_SCAN_PHY_1M))
+#endif /* CONFIG_BT_CTLR_PHY_CODED */
+	{
+		err = duration_period_update(scan, is_update_1m);
+		if (err) {
+			return err;
+		} else if (is_update_1m) {
+			return 0;
+		}
+	}
+
+	if (IS_ENABLED(CONFIG_BT_CTLR_ADV_EXT) &&
```

Comment:
This is inside an #if defined(CONFIG_BT_CTLR_ADV_EXT) block, no need to check for IS_ENABLED(CONFIG_BT_CTLR_ADV_EXT).